src/cascadia/TerminalApp/CascadiaSettings.h

```diff
@@ -79,7 +79,7 @@ class TerminalApp::CascadiaSettings final
     static std::filesystem::path GetSettingsPath();
     static std::filesystem::path GetDefaultSettingsPath();
 
-    const Profile* FindProfile(GUID profileGuid) const noexcept;
+    const winrt::TerminalApp::Profile FindProfile(GUID profileGuid) const noexcept;

```

Comment:
you would be having a much easier time with all this `GUID` business if you just changed the signatures to take `guid` instead...

Reply:
Eh. I have to do it at some point anyways. Good point.